sourcegraph/sourcegraph
# Code Insights: Support color for compute powered insight

Closes https://github.com/sourcegraph/sourcegraph/issues/38832
Fixes https://github.com/sourcegraph/sourcegraph/issues/39917

## Background 
Considering a given timeframe for Sourcegraph 4.0, we disable multi-series support for compute-powered insight. This PR 
- hides add a new series button in the creation UI
- adds colour picker to series creation form
- adjust the insight card layout for the compute-powered insight. Now it doesn't render legend block (see related comment https://github.com/sourcegraph/sourcegraph/pull/39805#issuecomment-1205879211) 

## Test plan
- Go to the compute insight creation UI 
- Create compute insight (make sure that you can't have more than one series) 
- See that this insight was created correctly, and you can see it on the dashboard page. 




## App preview:

- [Web](https://sg-web-vk-one-series-compute-insight.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-qgqlyenkco.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
